central/cloudsources/manager/manager.go

```diff
@@ -92,6 +97,16 @@ func (m *managerImpl) ShortCircuit() {
 	m.shortCircuitSignal.Signal()
 }
 
+func (m *managerImpl) MarkClusterSecured(id string) {
```

Comment:
Maybe matter of taste, but I feel like these two wrappers are unnecessary. Why not have one function to change the status?

Reply:
I didn't want callers to have to specify the status (i.e. `storage.DiscoveredCluster_STATUS_SECURED` or unsecured), but rather have this choice be made by the manager itself. Hence the two wrappers around the function, more or less to have it be simpler to deal with for the caller.